apps/src/aiTutor/views/AITutorFooter.tsx

```diff
@@ -110,11 +112,16 @@ const AITutorFooter: React.FC<AITutorFooterProps> = ({renderAITutor}) => {
           onChange={e => setUserMessage(e.target.value)}
           value={userMessage}
           disabled={disabled}
+          onKeyDown={e =>
+            e.key === 'Enter' && !userMessageIsEmpty
+              ? handleSubmit(ActionType.GENERAL_CHAT)
+              : null
```

Comment:
nit: I think swapping this to a dedicated handler instead of using a ternary to return null when we're not using the return value could be slightly cleaner, but logic looks good! 
```
const handleKeyPress = (e: React.KeyboardEvent) => {
    if (e.key === 'Enter' && userMessage.trim() !== '') {
      handleSubmit(ActionType.GENERAL_CHAT);
    }
  };
```